core/access-control-service/src/main/scala/edu/uci/ics/texera/service/resource/AccessControlResource.scala

```diff
@@ -38,11 +38,11 @@ object AccessControlResource extends LazyLogging {
 
   /**
     * Authorize the request based on the path and headers.
-   * @param uriInfo URI sent by Envoy or API Gateway
-   * @param headers HTTP headers sent by Envoy or API Gateway which include
-   *                headers sent by the client (browser)
-   * @return HTTP Response with appropriate status code and headers
-   */
+    * @param uriInfo URI sent by Envoy or API Gateway
```

Comment:
These format changes shouldn't be in this PR.

Reply:
I agreed. This change needs to be kept.

Reply:
Then you can do a separate PR to fix this formatting problem first?  We shouldn't include these changes in this PR.